Framework/include/Jet.h

```diff
@@ -1,13 +1,19 @@
 #ifndef JET_H
 #define JET_H
 
-inline bool passesPuIdMedium(float pt, float eta, float disc_value){
-     constexpr float cuts[4][4] = {
-        { 0.20,-0.56,-0.43,-0.38},
+inline bool passesPuIdMedium(float pt, float eta, float disc_value, std::string runYear){
+     constexpr float cuts[2][4][4] = {
```

Comment:
Is this object what is recommended? Can we make it a map or something?

Reply:
I'm sure we could. We just need a way to adjust our requirement based on eta and pt

Reply:
Ok at the very least it should be a member of the jet class. Looks like a global function at the moment.

Reply:
Also, can you make the array of arrays of arrays a member variable that is made by the constructor so we don't have to write it every event.

Reply:
Good call, I'll update to reflect